src/hooks/useCardAnalytics.ts

```diff
@@ -28,14 +29,17 @@ function isDirectAnswer(data: unknown): data is DirectAnswerData {
     (data as DirectAnswerData)?.type === DirectAnswerType.FieldValue;
 }
 
-export function useCardAnalytics(): (
-  cardResult: Result | DirectAnswerData, analyticsEventType: CardAnalyticsType
+export function useCardAnalytics<T = DefaultRawDataType>(): (
```

Comment:
is the reason a default is needed here for the type param because of when the hook is used with `DirectAnswerData`?

Reply:
thought we do export this so it would be good to have a default here, but we only export the callback wrapper functions (i.e. useCardAnalyticsCallback) which already have default so this is not needed. I can remove